### DISCUSSION TITLE
How to use Vuejs Devtools with Petite

### DISCUSSION THREAD

**Discussion by NickStees** [vuejs/petite-vue#143] [Q&A]
I LOVE petite-vue but **miss my [Vuejs Devtools](https://devtools.vuejs.org/guide/installation.html#standalone)** does anyone know how to get it to work? I checked [their issues](https://github.com/vuejs/devtools/issues?q=petite) and I don't see anyone having asked yet.

I am using the [ES module build](https://github.com/vuejs/petite-vue#:~:text=Or%2C%20use%20the-,es%20module%20build)  and I just can't get the dev tools to detect Vue petite.
`<script type="module">
              import { createApp } from 'https://unpkg.com/petite-vue?module'
</script>
`

**Comment by Abdillah** [comment]
It is unsupported currently, and I hope it will.
